cpg_utils/cloud.py

```diff
@@ -284,3 +287,25 @@ def _get_default_id_token_credentials(
             return current_credentials
 
     raise exceptions.DefaultCredentialsError(_HELP_MESSAGE)
+
+
+def get_cached_group_members(group) -> list[str]:
+    """
+    Get cached members of a group, based on the members_cache_location
+    """
+    group_name = group.split('@')[0]
+    config = get_config()
+
+    pathname = os.path.join(
+        config['infrastructure']['members_cache_location'], group_name + '-members.txt'
+    )
+
+    with AnyPath(pathname).open() as f:
+        return [line.strip() for line in f.readlines()]
+
+
+def check_member_in_cached_group_members(group, member) -> bool:
```

Comment:
`is_` makes the semantics a bit more clear IMO:

```suggestion
def is_member_in_cached_group(group, member) -> bool:
```